src/pymor/algorithms/timestepping.py

```diff
@@ -26,22 +26,36 @@
 class TimeStepper(ImmutableObject):
     """Interface for time-stepping algorithms.
 
-    Algorithms implementing this interface solve time-dependent problems
+    Algorithms implementing this interface solve time-dependent initial value problems
     of the form ::
 
-        M * d_t u + A(u, mu, t) = F(mu, t).
+        M(mu) * d_t u + A(u, mu, t) = F(mu, t),
+                         u(mu, t_0) = u_0(mu).
 
     Time-steppers used by |InstationaryModel| have to fulfill
     this interface.
     """
 
     @abstractmethod
```

Comment:
```suggestion
```
I would not require implementation.

Reply:
Then `solve` would not work. Or is that ok?

Reply:
Sorry, should have made my self clearer: If `estimate_time_step_number` raises the error, `solve` would need to catch it and live without the information.

Reply:
Aha, ok, makes sense.